content_restrictions/content_restrictions.py

```diff
@@ -352,22 +414,86 @@ def has_access_with_password(self, data, suffix=""):  # pylint: disable=unused-a
             "error_message": incorrect_password_explanation_text,
         }
 
-    # TO-DO: change this to create the scenarios you'd like to see in the
-    # workbench while developing your XBlock.
+    def set_seb_settings(self) -> None:
```

Comment:
same question here about static methods

Reply:
In this case if we need the instance, I don't see how we could use a static method.